src/features/sponsor/components/ListingWork.tsx

```diff
@@ -90,7 +90,13 @@ export function ListingWork() {
         >
           Get almost any kind of work done
         </Text>
-        <HStack pos="relative" align="start" gap={8} maxW={maxW}>
+        <HStack
+          pos="relative"
+          align="start"
+          gap={8}
+          overflow={'scroll'}
+          maxW={{ base: '100%' }}
+        >
```

Comment:
<img width="1317" alt="image" src="https://github.com/SuperteamDAO/earn/assets/67309069/e9106c47-157f-499b-8c17-d5cdadedb9e2">

show a little bit of the 4th testimonial so that someone knows its scrollable – or maybe let the container for testimonials get bigger than the container of the page to fit a bit of the 4th testimonial

Reply:
<img width="1353" alt="image" src="https://github.com/SuperteamDAO/earn/assets/91601706/2d4fea11-1c3d-45d9-ab51-95644bfe6301">
is this fine?

Reply:
but on larger screens, it looks weird
why dont you add an infinite horizontal scroll for them

Reply:
it should keep scrolling on its own

Reply:
ya got it.